spec/spec_helper.rb

```diff
@@ -11,6 +11,10 @@
 # Must be explicit as activerecord is optional dependency
 require 'active_record/railtie'
 
+unless File.exist?('config/database.yml')
```

Comment:
i would rather hardcode the paths than catching a generic `TypeError` because this can hide some other errors. Rails.application is not available at this moment because the application is initialized later a few lines below and throws the exception :(